src/mmw/js/src/analyze/templates/analyzeDescription.html

```diff
@@ -2,6 +2,15 @@
 <div class="layer-region"></div>
 <div class="analyze-source">
     {% if source %}
-        Source: {{ source }}
+        <span>Source: {{ source }}</span>
+        {% if helpText %}
+            <a class="help" data-toggle="popover" tabindex="0" data-html="true"
+               data-container="body" role="button" data-content="{{ helpText }}"
+               data-template="<div class='popover'><div class='pull-right'
+                                id='popover-close-button'><i class='fa fa-times' /></div><div
+                                class='popover-content'></div><div class='arrow'></div></div>">
+                <i class="fa fa-question-circle"></i>
```

Comment:
Consider having this match the appearance of the mapshed info popover

![screen shot 2017-07-20 at 9 13 58 am](https://user-images.githubusercontent.com/7633670/28419117-132905aa-6d2c-11e7-9185-2382ba420a8a.png)


(this one)
![screen shot 2017-07-20 at 8 54 36 am](https://user-images.githubusercontent.com/7633670/28419124-18bedf58-6d2c-11e7-87a8-60d5c261de73.png)